IncludeToolbox/IncludeFormatter/IncludeLineInfo.cs

```diff
@@ -146,6 +146,21 @@ public void UpdateTextWithIncludeContent()
             Delimiter1 = Delimiter0 + includeContent.Length + 1;
         }
 
```

Comment:
New function for the regex compare. If the option is default (false) just return includeContent. Otherwise, return the includeContent wrapped by delimiters.